repositories/testimonial-repository.js

```diff
@@ -21,11 +21,24 @@ const createTestimonial = async (req) => {
   return create;
 };
 
-const getTestimonials = async () => {
-  const list = await db.Testimonial.findAll({
+const getTestimonials = async (page) => {
+  //Find and count all testimonials
+  const list = await db.Testimonial.findAndCountAll({
+    limit: 10,
```

Comment:
I think it will be a good idea to make a variable out of this "10". It'll be easier to change the page size in the future.

Reply:
Just refactored the "limit" property into a constant. Thanks